egs/callhome_egyptian/s5/run.sh

```diff
@@ -125,15 +129,43 @@ steps/train_sat.sh  --cmd "$train_cmd" \
    exp/tri5a/graph data/dev exp/tri5a/decode_dev
 )&
 
-steps/decode_fmllr.sh --nj 25 --cmd "$decode_cmd" --config conf/decode.config \
-exp/tri5a/graph data/test exp/tri5a/decode_test
+(
+  steps/decode_fmllr.sh --nj 25 --cmd "$decode_cmd" --config conf/decode.config \
+  exp/tri5a/graph data/test exp/tri5a/decode_test
+  # Decode Supplement and H5
+  steps/decode_fmllr.sh --nj 25 --cmd "$decode_cmd" --config conf/decode.config \
+  exp/tri5a/graph data/sup exp/tri5a/decode_sup
+  steps/decode_fmllr.sh --nj 25 --cmd "$decode_cmd" --config conf/decode.config \
+  exp/tri5a/graph data/h5 exp/tri5a/decode_h5
+)&
+
+dnn_cpu_parallel_opts=(--minibatch-size 128 --max-change 10 --num-jobs-nnet 8 --num-threads 16 \
+                       --parallel-opts "-pe smp 16" --cmd "queue.pl -l arch=*64 -l mem_free=2G,ram_free=1G")
+dnn_gpu_parallel_opts=(--minibatch-size 512 --max-change 40 --num-jobs-nnet 4 --num-threads 1 \
+                       --parallel-opts "-l gpu=1" --cmd "queue.pl -l arch=*64 -l mem_free=2G,ram_free=1G")
 
-# Decode CALLHOME
-steps/decode_fmllr.sh --nj 25 --cmd "$decode_cmd" --config conf/decode.config \
-exp/tri5a/graph data/callhome_test exp/tri5a/decode_callhome_test
-steps/decode_fmllr.sh --nj 25 --cmd "$decode_cmd" --config conf/decode.config \
-exp/tri5a/graph data/callhome_dev exp/tri5a/decode_callhome_dev
-steps/decode_fmllr.sh --nj 25 --cmd "$decode_cmd" --config conf/decode.config \
-exp/tri5a/graph data/callhome_train exp/tri5a/decode_callhome_train
+steps/nnet2/train_pnorm_ensemble.sh \
```

Comment:
@noisychannel   is there a specific reason for using the ensemble 
training ? 

If interested in an improved acoustic model, you might want to look at steps/nnet3/train_tdnn.sh. Example usage of this script is available in egs/wsj/local/run_tdnn.sh. This would require nothing more than changing the train and decode commands. Most of the parameters are similar for the new scripts.